src/countervalues/mock.js

```diff
@@ -32,3 +33,93 @@ export const genStoreState = (pairs: Pair[]): CounterValuesState => {
   });
   return state;
 };
+
+export const getDailyRatesImplementation = (
+  getAPIBaseURL: () => string,
+  pairs: PollAPIPair[]
+) => {
+  const rates = {
+    USD: { BTC: { SIMEX: { latest: 0.00521587214628 } } },
+    BTC: {}
+  };
+
+  // For pairs not reflected here we will be falling back on 1  ¯\_(ツ)_/¯
+  const baseMockBTCRates = {
+    BCH: 0.06102,
+    BTG: 0.003239,
+    DASH: 0.02314614,
+    DCR: 0.004706,
+    DGB: 0.00000249,
+    DOGE: 5.392e-7,
+    ETH: 3.1983e-12,
+    ETC: 1.203e-13,
+    ZEN: 0.00135623,
+    KMD: 0.0002006,
+    LTC: 0.015603,
+    PPC: 0.011236,
+    PIVX: 0.0001792,
+    QTUM: 0.00055208,
+    XSN: 0.0000241,
+    XST: 0.0021319999999999998,
+    STRAT: 0.0002083,
+    VTC: 0.00009569,
+    VIA: 0.0001065,
+    XRP: 0.006208,
+    ZEC: 0.0133534
+  };
+
+  const arbitraryRateEpoch = 1555452000000; // 17th April 2019 00:00
+  const todayTimestamp = new Date().setHours(0, 0, 0, 0);
+
+  const hms = date =>
+    new Date(date.getTime() - date.getTimezoneOffset() * 60000)
+      .toISOString()
+      .split("T")[0];
+
+  const buildDays = from => {
+    const outputDays = {};
+    const baseRate = baseMockBTCRates[from] || 1;
+    const todayDate = new Date(todayTimestamp);
+    let date = new Date();
+
+    for (
+      date.setTime(todayTimestamp), date.setFullYear(date.getFullYear() - 3);
+      date.getTime() <= todayDate.getTime();
+      date.setDate(date.getDate() + 1)
+    ) {
+      const offset = (date.getTime() - arbitraryRateEpoch) / 864000000 / 180;
+      outputDays[todayDate - date ? hms(date) : "latest"] = Math.max(
+        baseRate * (1 + offset),
+        0
+      );
+    }
+    return outputDays;
+  };
+
+  return pairs.reduce((rates, pair) => {
+    if (pair.from === "BTC") return rates;
+    switch (pair.exchange) {
+      case "bad":
+        break;
+      case "ugly":
+        rates.BTC[pair.from] = { ugly: { latest: 0 } };
+        break;
+      default:
+        rates.BTC[pair.from] = {
+          [pair.exchange || "fallback"]: {
+            ...buildDays(pair.from)
+          }
+        };
+        break;
+    }
+    return rates;
+  }, rates);
+};
+
+export const fetchExchangesForPairImplementation = async () => [
+  { id: "good", name: "good", website: "#" },
+  { id: "ugly", name: "ugly", website: "#" },
+  { id: "bad", name: "bad", website: "#" }
+];
```

Comment:
please quickly document the behavior of each exchange

Reply:
makes sense